### PR TITLE
Added 2 more components

### PR DESCRIPTION
**Pull request by sebbb96**
I split some of the content in components and I look forward to add some functionalities to the buttons.

### CODE REVIEW

File: src/Components/TickerData.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const tickCoins = [
```
Comment (vekaev) [inline src/Components/TickerData.js]: better to move it to constants folder

File: src/Components/TickerCoins.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import "../App.css";
+import tickCoins from "./TickerData";
+const TickerCoins = () => {
+  const [coins, setCoins] = useState(tickCoins);
```
Comment (vekaev) [inline src/Components/TickerCoins.js]: Better to store state of the coins in App component and move delete logic there

File: src/App.js [hunk 3]
```diff
@@ -1,179 +1,17 @@
 import "./App.css";
 import Loading from "./Components/Loading";
-
+import Add from "./Components/AddCoins";
```
Comment (vekaev) [inline src/App.js:3]: https://stackoverflow.com/questions/48595120/react-exporting-and-importing-components

as I mentioned in other PRs

**Review by vekaev — COMMENTED** [review]
!

File: src/Components/TickerData.js (hunk 1 above)
Comment (sebbb96) [reply src/Components/TickerData.js]: Done

File: src/Components/TickerCoins.js (hunk 2 above)
Comment (sebbb96) [reply src/Components/TickerCoins.js]: Done

File: src/App.js (hunk 3 above)
Comment (sebbb96) [reply src/App.js:3]: Done and added tailwind to get rid of the huge css file

File: src/App.js [hunk 4]
```diff
@@ -1,179 +1,23 @@
-import "./App.css";
+import { useState } from "react";
 import Loading from "./Components/Loading";
-
+import Add from "./Components/AddCoins";
```
Comment (vekaev) [inline src/App.js:3]: import { Add, Loading } from "./components";

File: src/App.js [hunk 5]
```diff
@@ -1,179 +1,23 @@
...
-                viewBox="0 0 24 24"
-                fill="#ffffff"
-              >
-                <path d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"></path>
-              </svg>
-              Add
-            </button>
-          </section>
-
+          <Add />
```
Comment (vekaev) [inline src/App.js:17]: Naming?

File: src/Components/constants/TickerData.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+const tickCoins = [
```
Comment (vekaev) [inline src/Components/constants/TickerData.js:1]: extract constans from components folder

File: src/Components/TickerCoins.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
+const TickerCoins = ({ handleDelete, coins }) => {
+  return (
+    <>
+      <dl className="mt-5 grid grid-cols-1 gap-5 sm:grid-cols-3">
+        {coins.map((coin) => {
+          return (
+            <div
```
Comment (vekaev) [inline src/Components/TickerCoins.js:7]: move to component

**Pull request merged**